### PR TITLE
Updating homebrew instructions for Mac install

### PR DESCRIPTION
**Pull request by satb**

### CODE REVIEW

**Comment by Xosmond** [comment]
I was looking for this, Thanks mate.

File: source/guides/installing.html.md.erb [hunk 1]
```diff
@@ -25,7 +25,9 @@ Once the required dependencies are installed, set up Lucky for your system.
 
 Make sure you have [homebrew](https://brew.sh/), then run `brew install lucky`.
 
-If you are on macOS High Sierra you may need to add `export PKG_CONFIG_PATH=/usr/local/opt/openssl/lib/pkgconfig`
+* If you get an error saying formula cannot be found, make sure you run `brew tap luckyframework/homebrew-lucky`
```
Comment (paulcsmith) [inline source/guides/installing.html.md.erb:28]: I think at this point the guide might warrant having a list of items

* install Homebrew
* brew tap
* brew install
* if you’re on High Sierra export

I’m in my phone so sorry for the not so detailed comment. I hope this makes sense :)

**Comment by paulcsmith** [comment]
Thanks for catching this!

File: source/guides/installing.html.md.erb (hunk 1 above)
Comment (Xosmond) [reply source/guides/installing.html.md.erb:28]: I agree with you, the brew tap is  more an important step than a simple note

**Review by paulcsmith — COMMENTED** [review]
Thank you very much! This is much better. I'm going to merge this now and then make it into a list in a separate PR

**Pull request merged**